ssc/cmod_wind_landbosse.cpp

```
@@ -0,0 +1,354 @@
+}
+
+void cm_wind_landbosse::exec() {
+    // limit the input json through the process pip to only landbosse-required inputs
+    var_table input_data;
+    input_data.assign_match_case("weather_file_path", *m_vartab->lookup("wind_resource_filename"));
+    input_data.assign_match_case("distance_to_interconnect_mi", *m_vartab->lookup("distance_to_interconnect_mi"));
+    input_data.assign_match_case("interconnect_voltage_kV", *m_vartab->lookup("interconnect_voltage_kv"));
+    input_data.assign_match_case("depth", *m_vartab->lookup("depth"));
+    input_data.assign_match_case("rated_thrust_N", *m_vartab->lookup("rated_thrust_n"));
+    input_data.assign_match_case("labor_cost_multiplier", *m_vartab->lookup("labor_cost_multiplier"));
+    input_data.assign_match_case("gust_velocity_m_per_s", *m_vartab->lookup("gust_velocity_m_per_s"));
+    input_data.assign_match_case("wind_shear_exponent", *m_vartab->lookup("wind_resource_shear"));
+    input_data.assign_match_case("num_turbines", *m_vartab->lookup("num_turbines"));
+    input_data.assign_match_case("turbine_spacing_rotor_diameters", *m_vartab->lookup("turbine_spacing_rotor_diameters"));
+    input_data.assign_match_case("row_spacing_rotor_diameters", *m_vartab->lookup("row_spacing_rotor_diameters"));
+    input_data.assign_match_case("turbine_rating_MW", *m_vartab->lookup("turbine_rating_mw"));
+    input_data.assign_match_case("hub_height_meters", *m_vartab->lookup("wind_turbine_hub_ht"));
+    input_data.assign_match_case("rotor_diameter_m", *m_vartab->lookup("wind_turbine_rotor_diameter"));
+
+    std::string input_json = ssc_data_to_json(&input_data);
+	std::string input_dict_as_text = input_json;
+	std::replace(input_dict_as_text.begin(), input_dict_as_text.end(), '\"', '\'');
+
+    load_config();
+#ifdef __WINDOWS__
+	std::string output_json = call_python_module_windows(input_dict_as_text);
+#else
+    std::string output_json = call_python_module(input_dict_as_text);
+#endif
+
```

Should we throw a specific error if `output_json` is emtpy?  That condition could happen in the Windows function but isn't really handled.

---

Good point, we do that for the Unix function. Added.